aiojobs/_job.py

```diff
@@ -50,6 +56,20 @@ def pending(self) -> bool:
     def closed(self) -> bool:
         return self._closed
 
+    def get_name(self) -> str:
+        name = self._name
+        if sys.version_info >= (3, 8) and self._task:
+            name = self._task.get_name()
+        if name is None:
+            # Return generated default name
+            name = f"Job({self._coro})"
+        return name
```

Comment:
```suggestion
        return name or f"Job({self._coro})"
```

I think this is a little more concise.